<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockGlyphTable</class>
 <widget class="QDockWidget" name="DockGlyphTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glyph</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBoxGlyph">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Glyph &amp; Grid</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pageGlyphDimension">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>311</width>
             <height>240</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Glyph Dimension</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QSpinBox" name="spinBoxBitmapDimension"/>
            </item>
            <item row="1" column="0">
             <widget class="QListView" name="listView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageGlyph">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>158</width>
             <height>210</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Glyph</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QGroupBox" name="groupBitmapDimension">
              <property name="title">
               <string>Bitmap Dimension</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QSpinBox" name="glyphSize">
                 <property name="minimum">
                  <number>8</number>
                 </property>
                 <property name="maximum">
                  <number>640</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxBaseLines">
              <property name="title">
               <string>Left/Bottom baselines</string>
              </property>
              <layout class="QGridLayout" name="Left">
               <item row="0" column="1">
                <widget class="QPushButton" name="moveTop">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="main.qrc">
                   <normaloff>:/button/icons/up</normaloff>:/button/icons/up</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="moveLeft">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="main.qrc">
                   <normaloff>:/button/icons/left</normaloff>:/button/icons/left</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="resetOffset">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="main.qrc">
                   <normaloff>:/button/icons/center</normaloff>:/button/icons/center</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="moveRight">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="main.qrc">
                   <normaloff>:/button/icons/right</normaloff>:/button/icons/right</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="moveDown">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="main.qrc">
                   <normaloff>:/button/icons/down</normaloff>:/button/icons/down</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageGridDimension">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>311</width>
             <height>276</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Grid Dimension</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QPushButton" name="pushButtonGridMarginReset">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Grid Margins</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="1">
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Top</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QSpinBox" name="spinBoxGridMarginTop"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QGroupBox" name="groupBox">
                 <property name="title">
                  <string>Left</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QSpinBox" name="spinBoxGridMarginLeft"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QGroupBox" name="groupBoxMarginAll">
                 <property name="title">
                  <string>All</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QSpinBox" name="spinBoxMarginAll"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>Right</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QSpinBox" name="spinBoxGridMarginRight"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QGroupBox" name="groupBox_4">
                 <property name="title">
                  <string>Bottom</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QSpinBox" name="spinBoxGridMarginBottom"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>74</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Filter</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QTextEdit" name="textEditFilter"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageGlyphPreview">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>56</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Glyph Preview</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <widget class="QWidget" name="glyphPreview" native="true"/>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageStringPreview">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>56</height>
            </rect>
           </property>
           <attribute name="label">
            <string>String Preview</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <widget class="QLineEdit" name="lineEditPreview"/>
            </item>
            <item>
             <widget class="QWidget" name="stringPreview" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxCharacterTable">
       <property name="title">
        <string>Glyphs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTableView" name="tableViewGlyphs"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
